src/Components/Common/AzureComponent.cs

```diff
@@ -53,7 +53,15 @@ internal void AddClient(
         var configSection = builder.Configuration.GetSection(configurationSectionName);
```

Comment:
Can we remove the method

```
internal static string GetKeyedConfigurationSectionName(string key, string defaultConfigSectionName)
        => $"{defaultConfigSectionName}:{key}";
```

now? No one should be calling it.

Reply:
Yep, we can remove that method now.